syntax = "proto3";

package qtprotobuf.tests;

message SimpleIntMessage {
    int32 testFieldInt = 1;
}

message SimpleStringMessage {
    string testFieldString = 6;
}

message SimpleFloatMessage {
    float testFieldFloat = 7;
}

message SimpleDoubleMessage {
    double testFieldDouble = 8;
}
